tests/ubuntu_build_test.yaml

```diff
@@ -1,15 +1,11 @@
 schemaVersion: '2.0.0'
 commandTests:
-- name: 'path'
-  command: 'sh'
-  args: ['-c', 'echo $PATH']
-  expectedOutput: ['/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin']
-fileContentTests:
-- name: 'Ubuntu Distro Check'
-  expectedContents: ['.*ubuntu.*']
-  path: '/etc/os-release'
-fileExistenceTests:
-- name: 'Root'
-  path: '/'
-  shouldExist: true
-  permissions: 'drwxr-xr-x'
+- name: 'python on path'
+  command: 'which'
+  args: ['python']
+  expectedOutput: ['/usr/bin/python2.7']
```

Comment:
not totally sure but I think you need a newline here? these match regexes so you would need either newline or `.*` at the end

Reply:
These are passing locally as is.  The "echo $PATH" test that i used as a template doesn't append a newline even though the output has it.  Going  to leave as is as it appears to be working and is a little cleaner.

Reply:
ok yeah sounds good, if it's working don't change it just thought it might fail